raster/r.surf.random/testsuite/test_min_max.py

```diff
@@ -37,7 +35,6 @@ class MinMaxTestCase(TestCase):
     @classmethod
     def setUpClass(cls):
         """Ensures expected computational region"""
-        os.environ["GRASS_RANDOM_SEED"] = "42"
         # modifying region just for this script
         cls.use_temp_region()
         # Only 100,000,000 seem to resonably (not 100%) ensure that all values
```

Comment:
Do we still need this huge region?

Reply:
According to @wenzeslaus comment it sounds like we do, but I don't know for sure.

Reply:
I change the info test to test against fixed values for that seed.